lib/grn_ecmascript.lemon

```diff
@@ -400,6 +404,11 @@ relational_expression ::= relational_expression NEAR shift_expression. {
                           GRN_OP_PUSH, 1);
       grn_expr_take_obj(efsi->ctx, efsi->e, max_element_intervals);
       n_args++;
+      int min_interval;
+      GRN_INT32_POP(&efsi->min_interval_stack, min_interval);
+      grn_expr_append_const_int(efsi->ctx, efsi->e, min_interval,
+                                GRN_OP_PUSH, 1);
+      n_args++;
```

Comment:
Is it correct?
Does this mean if and only if `max_element_intervals` is specified, `min_interval` must be specified too?

Reply:
My intention is:

* Users must also specify `max_element_intervals` in order to specify `min_interval`.
  * They can omit the actual value of `max_element_intervals` like `*NP-1,0,,10 "hoge hoge"`
* Users can omit `min_interval`.
* if this is out of `if (max_element_intervals)`, `min_interval` could be the 3rd argument, but I couldn't get any good idea to decide to the 3rd argument is `min_interval` or `max_element_intervals`.
  * e.g.  `*NP-1,0,3 "hoge hoge"`, we can't decide that users intend `3` is `min_interval` or `max_element_intervals`.

Reply:
> `min_interval` must be specified too?

As comment above, we can omit `min_interval` in the current implementation.

For example, in the current implementation, if we specify the filter or query as below, `min_interval` is the default value `INT32_MIN`.

`*NP-1,0,2|1 "a 2 c"`

(I confirmed that with a debugger.)

Reply:
I will modify this as term of readability.
Please wait a while...

Reply:
Fixed.

Reply:
As far as I confirmed, it works fine. The second *N uses 3.
I have added tests for those cases.